### ISSUE TITLE
`cider cask missing` never discovers any missing casks

### ISSUE THREAD

**Issue by RogerThiede** [msanders/cider#38]
`cider cask missing` no longer appears to function after 84f5f9100f8e5e864bea05632ef8ffb5e39a4b4a / 4c3c8e672381a34ea7f9df13b00e99cba45f9f86.
It always reports `Everything up to date.` even when there are additional cask items that need to be added into `bootstrap.yaml` / `bootstrap.json`.

Example:

``` shell
 ~/.cider ‹master› » brew cask list
araxis-merge            google-chrome           keka                    macdown                 microsoft-office        p4v                     sourcetree              visual-studio-code    
dropbox                 grandperspective        komodo-ide              makemkv                 nvalt                   paragon-ntfs            vagrant                 vlc                   
 ~/.cider ‹master› » cider cask list
araxis-merge
dropbox
grandperspective
macdown
makemkv
nvalt
p4v
paragon-ntfs
sourcetree
vagrant
vlc
 ~/.cider ‹master› » cider cask missing
Everything up to date.
```

Expected Result:
`google-chrome`, `keka`, `komodo-ide`, `microsoft-office`, `visual-studio-code`

**Comment by RogerThiede** [comment]
@JMSwag, could #35 be an incomplete fix?

**Comment by JMSwag** [comment]
@RogerThiede It is an incomplete fix which #39 will address.
